FormSubmissionAndValidation2/src/components/DropdownBox.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import { ErrorMessage } from "./ErrorMessage";
+import { userFormConfig } from "../helper/userHelper";
+import React from "react";
+
+export const DropDownBox = React.memo(function DropDownBox({ fieldId, options, value, errorMessage, updateValue }) {
+  const label = userFormConfig[fieldId].label;
+
+  const optionElements = options.map((option) => {
+    return <option key={option} value={option} >{option}</option>
+  })
+  optionElements.splice(0,0, <option key="placeholder" value="" disabled hidden>Choose time zones</option>)
+
+  return <>
+    <div className="formRow">
```

Comment:
Nit, for classes, we prefer hyphenated names. form-row instead of formRow

Reply:
sure will make change